internal/ansi/progress.go

```diff
@@ -0,0 +1,29 @@
+package ansi
+
+import (
+	"errors"
+	"github.com/schollz/progressbar/v3"
+)
+
+func ProgressBar[T comparable](desc string, items []T, fn func(int, T) error) error {
+	switch len(items) {
+	case 0:
```

Comment:
Is zero a legitimate case?

Reply:
I think it is, but I'll add a test to prove it out.

Reply:
Shouldn't we just return nil directly in this case? Instead of a spinner with effectively an empty func. 🤔